### DISCUSSION TITLE
Allow setting default-value to an empty string instead of null

### DISCUSSION THREAD

**Discussion by rainmanhhh** [directus/directus#13826] [Feature Requests]
If I leave the input box blank, the default value will be NULL

**Comment by rijkvanzanten** [comment]
There's currently no way in the GUI to mark the default value as an empty string instead of null. You can set it directly in the database though.

I'll convert this Q&A to the feature request to allow that.

**Reply by sxsws** [reply]
I really haven't tried it, but would setting default value to a space, with the option to trim it enabled suffice? A nasty workaround though.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
